### ISSUE TITLE
type error in recent release of @jimp/plugins

### ISSUE THREAD

**Issue by tyler-johnson** [oliver-moran/jimp#864]
# Expected Behavior

Not have errors when building types.

# Current Behavior

Building types has this error:
```
node_modules/jimp/node_modules/@jimp/plugins/index.d.ts(3,20): error TS2307: Cannot find module '@jimp/plugin-circle'.

node_modules/jimp/node_modules/@jimp/plugins/index.d.ts(10,21): error TS2307: Cannot find module '@jimp/plugin-fisheye'.

node_modules/jimp/node_modules/@jimp/plugins/index.d.ts(20,20): error TS2307: Cannot find module '@jimp/plugin-shadow'.

node_modules/jimp/node_modules/@jimp/plugins/index.d.ts(21,23): error TS2307: Cannot find module '@jimp/plugin-threshold'.
```

# Failure Information (for bugs)

Looks like these dependencies are missing from package.json and are not installed.

## Steps to Reproduce

Build a TypeScript file that imports Jimp.

**Screenshots**

## Context

- Jimp Version: 0.9.7
- Operating System: Mac 10.15
- Node version: Node 13.11

**Comment by crutchcorn** [comment]
I'll be investigating tonight. Took a quick glance at the code and can't find anything immediately obviously missing. Makes me wonder if there's a `peerDep` issue

**Comment by crutchcorn** [comment]
Sure enough, it's a dependency issue. I'll resolve and have a PR out in no time :)

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in `v0.9.8` :rocket:
